src/js/view/view.js

```diff
@@ -157,6 +157,9 @@ function View(_api, _model) {
         const containerHeight = _model.get('containerHeight');
         const floating = _model.get('isFloating');
         if (containerWidth !== _lastWidth || containerHeight !== _lastHeight) {
+            if (!this.resizeListener) {
```

Comment:
i see this function being called only a few places in view, most of them pertaining to orientation/fullscreen changes along with floats. would this set up with a non-floating player in a normal scenario? i know there's a call in `init`, are the `_lastWidth`s (etc) not correct at the time of that init call and this causes it to get set up? want to be sure we're not missing any execution paths

Reply:
This always runs. The player fires "ready" only after `view.init` results in this code succeeding and firing a "resize" event. So what I'm doing here is deferring the creation of the resize listener until right before the ready event, when we know the player is in the DOM and we have valid `containerWidth` and `containerHeight` which the listener needs to setup properly.